### ISSUE TITLE
Can't update from 1.3.1 to 1.3.2 because RPMs are not in repo.

### ISSUE THREAD

**Issue by cameron-scrace** [openshift/openshift-ansible#3095]
I tried to use the upgrade playbook to move from 1.3.1 to 1.3.2 but Openshift version remained on 1.3.1. Looking at the repos enabled it appears that ansible playbook adds this repo: http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/
But the repo only has packages up to 1.3.1, are the 1.3.2 packages going to be added soon or am I doing something wrong?

Thanks.

**Comment by sdodson** [comment]
I'm not certain when those RPMs will be built. Assigned to @tdawson and @smarterclayton as they were sorting out whether or not there will be further 1.3 builds.

**Comment by tdawson** [comment]
I've been waiting for the 1.3.3 release to land to build the rpm, but I guess I should build the release from 1.3.2 while I'm waiting.

**Comment by tdawson** [comment]
The 1.3.3 rpm has been built on CentOS, has been flagged as release.  Hopefully it will go out to the mirrors on Monday.

**Comment by anroots** [comment]
On the subject of RPM availability: I found this thread, describing building `1.3.3` and `1.4` RPM-s for CentOS: https://www.spinics.net/lists/centos-devel/msg15601.html

Looks like they should be available next week and people can already test them out from the test repos:
```
yum --enablerepo=centos-openshift-origin-testing list origin openshift-ansible
```

Weakly related to https://github.com/openshift/openshift-ansible/issues/3027

**Comment by warmchang** [comment]
Pls check this repo, both origin-1.4.0 & origin-1.3.3 can be found:
https://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin/

**Comment by DanyC97** [comment]
@tdawson should we expect next week the `centos-openshift-origin` to get updated too ?

**Comment by tdawson** [comment]
I just checked the main repo, it is now 1.3.3.  That was updated yesterday.
origin 1.4  and openshift-ansible 3.4 is still going through testing, and both rpms need to be rebuilt.  So for 1.4, next week will be the earliest I can think of.

**Comment by tdawson** [comment]
origin 1.4.1 is now in the centos paas testing repo https://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin/
We are expecting them to be released sometime next week, unless we have major issues with tests.

Since this Issue was about 1.3, and 1.3.3 is in the released centos repo, I am closing this issue.

**Comment by warmchang** [comment]
Hi @tdawson , It look like the RPM package of OCP version prior to the community version released. Am I right?

[Red Hat OpenShift Container Platform 3.4 is now available](https://access.redhat.com/announcements/2861871)

**Comment by tdawson** [comment]
It depends on your definition of "community version".
Did it release before CentOS Paas SIG released theirs.  Definitely.
Did it release before OpenShift Origin released 1.4, I'm not totally sure.  They like to time them pretty close.  I believe OCP 3.4 and OpenShift Origin 1.4 were released on the same week.  Which was first, I haven't checked.

**Comment by warmchang** [comment]
@tdawson ok, I got it, thank you! :+1:
